cmd/create/cluster/cmd.go

```diff
@@ -418,18 +431,27 @@ func run(cmd *cobra.Command, _ []string) {
 
 	clusterID := cluster.ID()
 	clusterName := cluster.Name()
-	reporter.Infof("Creating cluster with identifier '%s' and name '%s'", clusterID, clusterName)
-	reporter.Infof("To view list of clusters and their status, run 'moactl list clusters'")
-
+	reporter.Debugf("Creating cluster with identifier '%s' and name '%s'", clusterID, clusterName)
```

Comment:
This line does seem a bit misplaces even as debug since there's no action between this and "Cluster has been created"
If this moved to L425 it could stay on info level

The flow from a user perspective would be 
* creating cluster
* short wait
* cluster has been created 

or in case of failure
* creating cluster
* short wait
* failed to create cluster

Reply:
Good catch, thanks! I've removed that line, as we don't have the cluster ID prior to creating it